src/SugarChat.WebApi/Controllers/UserController.cs

```diff
@@ -39,6 +40,36 @@ public async Task<IActionResult> UpdateMyProfile(UpdateUserCommand command)
             return Ok(response);
         }
 
+        [Route("addUser"), HttpPost]
```

Comment:
the route of "api/user/adduser" seems a little redundent on "user".

Reply:
commit